src/main/scala/activemq/cli/command/Commands.scala

```diff
@@ -144,9 +145,12 @@ abstract class Commands extends PrintStackTraceExecutionProcessor {
     ActiveMQCLI.broker match {
       case Some(matched) ⇒
         val jmxurls = matched.jmxurl.split(",")
-        jmxurls.foreach { url ⇒
+        // jmxurls.foreach { url ⇒
+        for (i ← 0 to jmxurls.size - 1) {
+          var url = jmxurls(i)
+          println("Current URL: " + url)
           if (!isConnected) {
-            // println("Attempting to connect to URL: " + url)
+            println("Attempting to connect to URL: " + url)
```

Comment:
Looks as if these println statements are added for debugging purposes. Please remove them.